src/renderer/modules/settings/screens/formBuilder.ts

```diff
@@ -312,7 +312,7 @@ export function getFormSections(repositoryData: TRepositoryData): IConfigSection
           type: 'file-path',
           hide: repositoryData[TLS_AUTH_TYPE_CONFIG] !== TlsAuthType.ServerClientCertAuth_Pfx,
           infoText: translate('info_text_pfx_file'),
-          required: repositoryData[TLS_AUTH_TYPE_CONFIG] === TlsAuthType.ServerClientCertAuth_Pfx,
+          required: false, // repositoryData[TLS_AUTH_TYPE_CONFIG] === TlsAuthType.ServerClientCertAuth_Pfx,
```

Comment:
Hi Manuel, thank you for the MR.

Actually, the logic we have implemented says that it is required if you choose the p12 certificate for authentication, and we think that this is the right way.

If you select p12 file authentication, then you really need to upload a p12 file.